src/game/server/entities/character.cpp

```diff
@@ -2151,6 +2190,33 @@ void CCharacter::Tick()
 				NULL
 			);
 		}
+		
+		
+		CWhiteHole* pCurrentWhiteHole = NULL;
+		for(CWhiteHole *pWhiteHole = (CWhiteHole*) GameWorld()->FindFirst(CGameWorld::ENTTYPE_WHITE_HOLE); pWhiteHole; pWhiteHole = (CWhiteHole*) pWhiteHole->TypeNext())
+		{
+			if(pWhiteHole->m_Owner == m_pPlayer->GetCID())
+			{
+				pCurrentWhiteHole = pWhiteHole;
+				break;
+			}
+		}
+		
+		
+		
+		//POTENTIAL CONFLICT WITH MINE BROADCAST --> make this Broadcast higher priority
```

Comment:
So if someone would keep a white hole for later use it basically hides the mine counter? What about displaying both information or playing a sound when gaining one?

Reply:
If you gain a white hole, you only get a chat message and a visual indicator around your tee. So there is no problem with keeping one for later.
Its just when you use one, it will tell you how many seconds are left until the hole collapses and i think this counter will overwrite the mine counter.

Reply:
There is a broadcast for mines placed by scientist like there is one for the white hole.
Both activated, one overwrite the text of the other. But we can implement a chat message for the white hole instead, which in comparison to the broadcast won't update as much.